src/WSStatsHooks.php

```diff
@@ -61,47 +64,49 @@ public static function getConfigSetting( string $name ) {
 	 *
 	 * @return mixed
 	 */
-	public static function getPageTitleFromID( $id ) {
+	public static function getPageTitleFromID( $id ) : mixed {
```

Comment:
Merk op: mixed is een php8.0 pseudotype. Je moet dus ook in `composer.json` de `require: php: "^7.0"` veranderen naar `"^8.0"` of `">=8.0"`.